man/repart.d.xml

```diff
@@ -217,6 +217,15 @@
         automatically used.</para></listitem>
       </varlistentry>
 
+      <varlistentry>
+        <term><varname>Uuid=</varname></term>
```

Comment:
we probably should spell this out as UUID=, it's how we usually do it with acronyms across the config files in our codebase.